src/DependencyInjection/RuudkPaymentMollieExtension.php

```diff
@@ -44,11 +44,9 @@ protected function addFormType(array $config, ContainerBuilder $container, $meth
         $mollieMethod = 'mollie_' . $method;
 
         $definition = new Definition();
-        $definition->setClass('%ruudk_payment_mollie.form.mollie_type.class%');
-        $definition->addArgument($mollieMethod);
```

Comment:
Thinking about it, this might remove compatibility with Symfony < 3.0, which might not be what we want?